examples/all-clusters-app/all-clusters-common/src/binding-handler.cpp

```diff
@@ -102,10 +103,20 @@ static void BoundDeviceChangedHandler(const EmberBindingTableEntry & binding, ch
     }
 }
 
-CHIP_ERROR InitBindingHandlers()
+static void InitBindingHandlerInternal(intptr_t arg)
 {
-    chip::BindingManager::GetInstance().SetAppServer(&chip::Server::GetInstance());
+    auto & server = chip::Server::GetInstance();
+    chip::BindingManager::GetInstance().Init(
+        { &server.GetFabricTable(), server.GetCASESessionManager(), &server.GetPersistentStorage() });
     chip::BindingManager::GetInstance().RegisterBoundDeviceChangedHandler(BoundDeviceChangedHandler);
+}
+
+CHIP_ERROR InitBindingHandlers()
+{
+    // The initialization of binding manager will try establishing connection with unicast peers
```

Comment:
This is probably OK for now, but please file a TODO to sort out how this should actually work.